stac_fastapi/types/stac_fastapi/types/config.py

```diff
@@ -30,11 +31,7 @@ class ApiSettings(BaseSettings):
     openapi_url: str = "/api"
     docs_url: str = "/api.html"
 
-    class Config:
-        """Model config (https://pydantic-docs.helpmanual.io/usage/model_config/)."""
-
-        extra = "allow"
-        env_file = ".env"
+    model_config = SettingsConfigDict(env_file=".env", extra="allow")
```

Comment:
I think it would be nice to have a small section in the readme explaining how to set the response validation.

Reply:
I added some language to the Readme

Reply:
Looks great. Thank you